src/App.tsx

```diff
@@ -172,6 +173,8 @@ const AppRoutes = ({ base }: AppProps) => {
                   </Route>
                   <Route path="subjects" element={<FavoriteSubjectsPage />} />
                 </Route>
+                <Route path="about/:slug" element={<AboutPage />} />
```

Comment:
Trengs denne? Hadde håpt vi kunne ha slug direkte på rota, men det krever vel at vi skriver om slik at vi har en fallback-side som tar alle pather som ikkje er definert andre steder. Kanskje litt i overkant.

Reply:
Tror det kommer til å bli litt seigt, ja. Det krever vel egentlig at vi alltid kjører et kall for article-api, som jeg synes blir litt overkill. Løsningen jeg gikk for her støtter at vi har flere dynamiske menyer, slik at vi er klare dersom de plutselig vil ha noe annet enn "Om NDLA" også. Da synes jeg det gir mening å ha logiske grupperinger